corehq/motech/repeaters/models.py

```diff
@@ -620,6 +622,7 @@ def handle_success(self, response):
             success_response=self._format_response(response) if response else None,
             next_check=None,
             succeeded=True,
+            payload=self.get_payload(),
```

Comment:
Are we doing this for all repeaters then? This would add quite some load on DB. Can we restrict to only those repeaters where this is needed?

Reply:
I have the same conservative instinct here, @proteusvacuum do you think we're making a fuss over something pretty small? I could see storing all form xml/json in duplicate inside repeat records in couch being a non-trivial hit

Reply:
@mkangia @dannyroberts I've turned this into an `info` field on the `RepeatRecordAttempt` which each Repeater is now free to use as they wish. I've currently only enabled it for the repeaters that we need this functionality for. Does that seem like a reasonable tradeoff?

Reply:
Sounds good.

nit: i think we can just rename this to `payload_info` since its payload anyway.